infra-as-code/bicep/machinelearning.bicep

```diff
@@ -159,28 +158,22 @@ resource aiHub 'Microsoft.MachineLearningServices/workspaces@2024-07-01-preview'
     friendlyName: 'Azure OpenAI Chat Hub'
     description: 'Hub to support the Microsoft Learn Azure OpenAI basic chat implementation. https://learn.microsoft.com/azure/architecture/ai-ml/architecture/basic-openai-e2e-chat'
     publicNetworkAccess: 'Enabled' // Production readiness change: The "Baseline" architecture adds ingress and egress network control over this "Basic" implementation.
-    ipAllowlist: []
-    serverlessComputeSettings: null
-    enableServiceSideCMKEncryption: false
     managedNetwork: {
-      isolationMode: 'Disabled' // Production readiness change: The "Baseline" architecture adds ingress and egress network control over this "Basic" implementation.
+      isolationMode: 'Disabled'
     }
-    allowRoleAssignmentOnRG: false // Require role assignments at the resource level.
     v1LegacyMode: false
     workspaceHubConfig: {
-      defaultWorkspaceResourceGroup: resourceGroup().id  // Setting this to the same resource group as the workspace
+      defaultWorkspaceResourceGroup: resourceGroup().id // Setting this to the same resource group as the workspace
     }
 
     // Default settings for projects
     storageAccount: aiStudioStorageAccount.id
     containerRegistry: containerRegistry.id
-    systemDatastoresAuthMode: 'identity'
-    enableSoftwareBillOfMaterials: true
     enableDataIsolation: true
     keyVault: keyVault.id
-    applicationInsights: applicationInsights.id
     hbiWorkspace: false
-    imageBuildCompute: null
+    applicationInsights: applicationInsights.id
+    discoveryUrl: 'https://${location}.api.azureml.ms/discovery'

```

Comment:
Why did discoveryUrl need to be added?

Reply:
I brought this over from the e2e-baseline code where I was experimenting a lot to get it aligned and working with the latest api version.

Reply:
Let's remove it. I think that's a RO property.

Reply:
discovery URL is still present.